pkg/bundle/internal/target/target.go

```diff
@@ -455,3 +487,17 @@ func TrustBundleHash(data []byte, additionalFormats *trustapi.AdditionalFormats)
 
 	return hex.EncodeToString(hashValue[:])
 }
+
+// mapsFromBundle returns the annotations and labels maps from a template ensuring the maps are not nil.
+func mapsFromBundle(targetMetadata *trustapi.TargetMetadata) (map[string]string, map[string]string) {
+	annotations := map[string]string{}
+	labels := map[string]string{}
+
+	if targetMetadata != nil && targetMetadata.Annotations != nil {
+		annotations = targetMetadata.Annotations
+	}
+	if targetMetadata != nil && targetMetadata.Labels != nil {
+		labels = targetMetadata.Labels
+	}
+	return annotations, labels
+}
```

Comment:
```suggestion
```

Instead of this helper function, I would suggest creating nil-safe getters for `Labels` and `Annotations` on the `TargetMetadata` struct.